aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -384,7 +384,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> awaitDBInstanceAvailable
                 .translateToServiceRequest(Function.identity())
                 .backoffDelay(config.getBackoff())
                 .makeServiceCall(NOOP_CALL)
-                .stabilize((request, response, proxyInvocation, model, context) -> isDBInstanceStabilizedAfterUpdate(proxyInvocation, model))
+                .stabilize((request, response, proxyInvocation, model, context) -> isDBInstanceStabilizedAfterReboot(proxyInvocation, model))
```

Comment:
nit: method invoking this call is called `awaitDBInstanceAvailableStatus`, which might confuse us in the future. Maybe it is worth indicating that this method should only be called after reboot?

Reply:
Good call. Fixed now.